ml-agents/mlagents/trainers/ppo/optimizer.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, policy: TFPolicy, trainer_params: TrainerSettings):
 
                 self.stream_names = list(self.reward_signals.keys())
 
-                self.tf_optimizer: Optional[tf.train.AdamOptimizer] = None
+                self.tf_optimizer_op: Optional[tf.train.Optimizer] = None
```

Comment:
Slightly more generic type, and rename to make clear that it's not a TFOptimizer instance.

Reply:
👍